webextensions/api/identity.json

```diff
@@ -12,9 +12,15 @@
               "edge": {
                 "version_added": "79"
               },
-              "firefox": {
-                "version_added": "53"
-              },
+              "firefox": [
+                {
+                  "version_added": "53"
+                },
+                {
+                  "version_added": "86",
+                  "notes": "The <code>redirect_url</code> parameter now supports a loopback address, see <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/identity#getting_the_redirect_url'>Getting the redirect URL</a> for details."
```

Comment:
The `redirect_url` property belongs to the `launchWebAuthFlow` method, not here.